doctr/models/builder.py

```diff
@@ -272,7 +272,9 @@ def __call__(
         self,
         boxes: List[np.ndarray],
         text_preds: List[List[Tuple[str, float]]],
-        page_shapes: List[Tuple[int, int]]
+        page_shapes: List[Tuple[int, int]],
+        orientations: Optional[List[Dict[str, Any]]] = [],
```

Comment:
Two things here:
1. this is typed as `Optional`, so the default would be `None`
2. having complex data types such as tuple or list as default of a function have a few underlying issues in rare cases. To make it short, let's not do this :sweat_smile:

Reply:
Yes my bad :+1: 
Done and pushed.